### PR TITLE
deployments/physical/aws: clean

### PR DESCRIPTION
**Pull request by yorickvP**
I haven't tested this

### CODE REVIEW

File: deployments/physical/aws/default.nix [hunk 1]
```diff
@@ -48,26 +48,27 @@
...
+      usePublicDNSName = !production;
       hostName =  "${config.networking.hostName}.net.${realDomain}";
     };
 
     deployment.targetEnv = "ec2";
 
-    deployment.ec2 = {
+    deployment.ec2 = with resources; {
       inherit region accessKeyId;
+      instanceType = "c5.xlarge";
```
Comment (mkaito) [inline deployments/physical/aws/default.nix]: I mean, yeah, it's DRY, but maybe at least make it `mkDefault`.

**Review by mkaito — CHANGES_REQUESTED** [review]
99% good

**Comment by yorickvP** [comment]
Apparently, you are supposed to use `subnetGroupIds` if you use `subnets`, which is dumb.

**Comment by yorickvP** [comment]
> 20:23 <aminechikhaoui> yeah we should fix that I think but for now I think it's that way for historical reasons
> 20:23 <aminechikhaoui> it was that way when I started using nixops :D

**Comment by mkaito** [comment]
If that assertion is wrong, should it not be removed?

El 7 de septiembre de 2018 19:25:21 GMT+01:00, Yorick <notifications@github.com> escribió:
>> 20:23 <aminechikhaoui> yeah we should fix that I think but for now I
>think it's that way for historical reasons
>> 20:23 <aminechikhaoui> it was that way when I started using nixops :D
>
>
>-- 
>You are receiving this because your review was requested.
>Reply to this email directly or view it on GitHub:
>https://github.com/DisciplinaOU/disciplina-profiles/pull/1#issuecomment-419525867

-- 
Enviado desde mi dispositivo Android con K-9 Mail. Por favor, disculpa mi brevedad.

**Comment by mkaito** [comment]
Update from Slack: nixops is wrong and should be fixed, or we should just fix this PR and merge.

Update update: I'm fixing the PR.

**Review by mkaito — APPROVED** [review]

**Pull request merged**